### DISCUSSION TITLE
Allow Quarto's MacOS installer to run without Rosetta

### DISCUSSION THREAD

**Discussion by kingo55** [quarto-dev/quarto-cli#8731] [Feature Requests]
### Description

I've been curious to try Quarto, but it currently doesn't support Apple Silicone without installing Rosetta, so I may end up trying it on an older Intel machine. If this was built for Apple ARM-based CPUs, it would significantly lower the barrier to entry for trying Quarto for other users of Apple Silicone.

Currently Knitr and Pandoc support Apple Silicone, so from my limited knowledge, it sounds like it might be doable with minimal configuration.

Edit: The issue appears to be the pkg installer and not Quarto's binaries itself.

**Comment by cscheid** [comment]
Have you tried running Quarto? I'm fairly sure our latest 1.4 releases support Apple Silicon; the last dependency was Pandoc, but they ship ARM binaries now, and so we use then.

**Reply by kingo55** [reply]
Thanks for the quick response. Yes, I believe you're right - in the tar archive I see a universal binary and arm ones too:

<img width="842" alt="Screenshot 2024-02-14 at 17 20 51" src="https://github.com/quarto-dev/quarto-cli/assets/2361388/57ba2d5f-b6c6-4e5e-9ee9-633da38ac666">

And the `quarto` binary seems to run on my Apple M2 - knowing this I may install it manually: 

```
$ ./quarto -v
1.4.549
```

However when I run the installation .pkg file, it throws this message up:

<img width="1199" alt="Screenshot 2024-02-14 at 17 17 03" src="https://github.com/quarto-dev/quarto-cli/assets/2361388/f92d3216-ca44-4793-a496-c92a6deba9a1">

Denying this prompt closes the installer with:

<img width="294" alt="Screenshot 2024-02-14 at 17 17 55" src="https://github.com/quarto-dev/quarto-cli/assets/2361388/c76c1861-7ef6-46af-a650-904e9ec70ed2">

I'll rename this issue more appropriately. Thanks again!

**Reply by dragonstyle** [reply]
Thanks for letting us know! I think I found our omission in the MacOS installer and I have a fresh build of it on the way. My Mac has Rosetta already, so if you have a moment to test the installer (in our pre-release initially) here:

https://github.com/quarto-dev/quarto-cli/releases/tag/v1.5.14

I would really appreciate it (if not, I understand and we'll figure out another way). Assuming this works, we will back-port to our next patch release of Quarto 1.4, which is imminent so you don't need to complete installation of Quarto 1.5 if you'd rather stick with a full release.
